jdaviz/configs/specviz/plugins/unit_conversion/tests/test_unit_conversion.py

```diff
@@ -121,7 +88,6 @@ def test_no_spec_no_flux_uncert_unit_exp_none(specviz_app, spectrum1d):
 
 
 def test_no_spec_flux_no_uncert(specviz_app, spectrum1d):
-    np.random.seed(42)
```

Comment:
Funnily enough, I don't think you need to set a seed here, unless I have missed something.